src/features/landing/components/CardItem/CardItem.tsx

```diff
@@ -0,0 +1,68 @@
+'use client';
+
+import { Grid, Typography } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface CardItemInterface {
+  item: {
+    id: string;
+    imageSrc: string;
+    title: string;
+    text: string;
+  };
+  type: 'WhyUs' | 'Possibilities';
+}
+
+function CardItem({ item, type }: CardItemInterface) {
+  const { t } = useTranslation();
+
+  return (
+    <div>
+      {type === 'WhyUs' && (
```

Comment:
its looks strange, this component must be reusable so need rework this logic, 'WhyUs' is not a type its looks like a name for something
you can create main cardItem component which can render different card by type, for example 

cardItem
- cardItemSmall
- cardItemBig

and a type can be a size of card or something another

Reply:
In progress